GitUI/Shells/ConEmuControlExtensions.cs

```diff
@@ -16,14 +16,17 @@ public static void ChangeFolder(this ConEmuControl? terminal, IShellDescriptor?
             switch (shell.Name)
             {
                 case BashShell.ShellName:
+                    // Use a ConEmu macro to send the sequence for clearing the bash command line
                     terminal.RunningSession.BeginGuiMacro("Keys").WithParam("^A").WithParam("^K").ExecuteSync();
-                    terminal.RunningSession.WriteInputTextAsync(command + Environment.NewLine);
+                    terminal.RunningSession.WriteInputTextAsync(command);
                     break;
 
                 default:
-                    terminal.RunningSession.WriteInputTextAsync($"\x1B{command}{Environment.NewLine}");
+                    terminal.RunningSession.WriteInputTextAsync($"\x1B{command}");
```

Comment:
Which term is which?

Reply:
🤔hummm.... I don't get it.

I have remove the new line because it is no more working in ConEmu and replace by sending the <kbd>Enter</kbd> key to run the command...

Reply:
The first case is BashShell.ShellName, no `\x1B`
Is the macro invoked in both situations, what is the difference?
How to test both paths?

Reply:
The first case if for bash where we use a macro because it doesn't react like other shells (cmd, powershell, pwsh) that clean the command line when you print the <kbd>Esc</kbd> ( `\x1B` ) char.

>Is the macro invoked in both situations, what is the difference?

Then the macro to send `Enter` is used to press ENter in both cases (now that printing new line doesn't work anymore)

>How to test both paths?

Switching the default shell 
![image](https://user-images.githubusercontent.com/460196/210904261-92995d2e-1fb8-45a2-8fdc-f9eb1374c86a.png)